### PR TITLE
pelux.conf: set DISTRO_VERSION to 4.0

### PR DESCRIPTION
**Pull request by sashko**
Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

File: conf/distro/pelux.conf [hunk 1]
```diff
@@ -9,7 +9,7 @@ require conf/distro/poky.conf
 MAINTAINER = "Gordan Markuš <gordan.markus@pelagicore.com>"
```
Comment (thenor) [inline conf/distro/pelux.conf]: We should remove the maintainer here also.

File: conf/distro/pelux.conf [hunk 2]
```diff
@@ -9,7 +9,7 @@ require conf/distro/poky.conf
...
 DISTRO = "pelux"
 DISTRO_NAME = "PELUX"
-DISTRO_VERSION = "3.0"
+DISTRO_VERSION = "4.0"
```
Comment (thenor) [inline conf/distro/pelux.conf]: I feel it is too early to change this version number.

Comment (sashko) [reply conf/distro/pelux.conf]: why?

File: conf/distro/pelux.conf (hunk 1 above)
Comment (sashko) [reply conf/distro/pelux.conf]: yes, will do in a separate commit

Comment (sashko) [reply conf/distro/pelux.conf]: actually, I believe this is an obligatory field and yocto-layer-check will fail, but let me check first...

Comment (sashko) [reply conf/distro/pelux.conf]: nah, it's cool; will remove later.

File: conf/distro/pelux.conf (hunk 2 above)
Comment (thenor) [reply conf/distro/pelux.conf]: We're not on version 4. It's one thing for SWF to point to "4.0-alpha" but as far as I know this doesn't mention anything about a preliminary/alpha release.

Comment (sashko) [reply conf/distro/pelux.conf]: of course, we're on 4.0 dev.

**Comment by fhajredini** [comment]
Regarding the recent PR to go on 4.0, i disagree as well. This is the first time that i hear about 4.0 dev and so on.. We can hold this PR open until we release 4.0

**Comment by fhajredini** [comment]
Are we keeping this PR until we release 4.0 or shall we close this?

**Comment by sashko** [comment]
I strongly belive we should clearly distinguish released versions from the development ones. Perhaps setting the version to `4.0` just yet isn't the best idea, but the must be something that indicates this is not `3.0` because it is in fact not.

**Pull request closed without merging**